client/tileset/tilespec.h

```diff
@@ -137,6 +137,9 @@ tileset_get_layers(const struct tileset *t);
 // Gfx support
 QPixmap *load_sprite(struct tileset *t, const QStringList &possible_names,
                      bool required, bool verbose = true);
+QPixmap *terrain_canvas(struct terrain *terrain,
+                        const struct extra_type *resource = nullptr,
+                        enum extra_cause cause = EC_COUNT);
```

Comment:
Nothing against moving it here, but it looks like you're not using this function in the end?

Reply:
I did end up not using it due to the bug